src/main/scala/is/hail/expr/AST.scala

```diff
@@ -785,6 +807,15 @@ case class ApplyMethod(posn: Position, lhs: AST, method: String, args: Array[AST
         `type` = FunctionRegistry.lookupMethodReturnType(it, funType +: rest.map(_.`type`), method)
           .valueOr(x => parseError(x.message))
 
+      case (tt: TTuple, "[]", Array(Const(_, v, t))) =>
+        val idx = t match {
+          case TInt32(_) => v.asInstanceOf[Int]
+          case _ => fatal(s"Tuple index must have type `tint32'. Found ${ t.toString }.")
```

Comment:
Don't use the lower-case stuff yet -- let's keep it consistent and do it all at once.

Reply:
though this will never happen, actually. Just make it an unchecked match.